modules/statics/src/coins.ts

```diff
@@ -292,7 +292,7 @@ export const coins = CoinMap.fromCoins([
   erc20('snc', 'SunContract', 18, '0xf4134146af2d511dd5ea8cdb1c4ac88c57d60404', UnderlyingAsset.SNC),
   erc20('snov', 'Snovio', 18, '0xbdc5bac39dbe132b1e030e898ae3830017d7d969', UnderlyingAsset.SNOV),
   erc20('snt', 'Status Network Token', 18, '0x744d70fdbe2ba4cf95131626614a1763df805b9e', UnderlyingAsset.SNT),
-  erc20('snx', 'Synthetix Network', 18, '0xc011a72400e58ecd99ee497cf89e3775d4bd732f', UnderlyingAsset.SNX),
+  erc20('snx', 'Synthetix Network', 18, '0xC011a73ee8576Fb46F5E1c5751cA3B9Fe0af2a6F', UnderlyingAsset.SNX),
```

Comment:
Please change to all-lowercase instead of EIP-55 casing. Unfortunately some of our components expect it that way

Reply:
No worries - done.